deployment/README.md

```diff
@@ -0,0 +1,136 @@
+# CCNP Deployment Guide
+
+
+## Create TD
+
+CCNP is designed for collecting confidential computing primitives from confidential virtual machines. It's designed to run as daemon set on confidential virtual machines nodes, such as Intel Trust Domain (TD), in a Kubernetes cluster.
+You can use [cvm image rewriter](../tools/cvm-image-rewriter/README.md) to prepare a TD enlightened guest image and start a TD using [qemu-test.sh](../tools/cvm-image-rewriter/qemu-test.sh)
+
+## Prepare a K8S cluster with TD as worker nodes
+
+You can either create a K8S cluster in the TD or let the TD join an existing K8S cluster. Please choose one of the following step to make sure the K8S cluster is prepared with the TD running in it. CCNP will be deployed on the TD later.
+
+### Option 1: Create a K8S cluster on the TD
+After TDs are started, users need to setup a K8S cluster in the TDs. Please refer to the [k8s official documentation](https://kubernetes.io/docs/home/) for detailed steps. 
+
+_NOTE: If the cluster has only one node (master node), the taint on the node needs to be removed._
+
+### Option 2: Add the TD to an existing K8S cluster
+TBD
+
+## Deploy CCNP
+
+- [image-manager.sh](../deployment/script/image-manager.sh): The tool will build 4 images and push them to remote registry if required.
+- [deploy-ccnp.sh](../deployment/script/deploy-ccnp.sh): The tool will deploy CCNP services as DaemonSet on TDs in the K8S cluster.
+- [deploy-and-exec-ccnp-example.sh](../deployment/script/deploy-and-exec-ccnp-example.sh): The tool will deploy an example pod and show getting event logs, measurement and quote using CCNP in the pod.
```

Comment:
deploy an example pod and show getting event logs, measurement using CCNP and perform verification in the pod.

Reply:
updated